ansible_collections/arista/avd/roles/eos_designs/doc/l3ls-evpn/network-services.md

```diff
@@ -158,8 +161,10 @@ tenants:
 
             # ip virtual-router address
             # note, also requires an IP address to be configured on the SVI where it is applied.
-            # Optional
-            ip_virtual_router_address: < IPv4_address/Mask >
+            # Optional, don't assign mask else that IP will be rejected
```

Comment:
Are you sure it will be rejected? The mask will be removed by the filter, but not the IP will survive.
I think we should actually remove that filter in the templates, so we accept mask on this as well. There are valid use cases for that, and it will improve our data model coverage.